virttest/utils_secret.py

```diff
@@ -0,0 +1,52 @@
+"""
+Secret related utility functions
+"""
+
+import logging
+
+from virttest import virsh
+
+
+def get_secret_list(remote_virsh=None):
+    """
+    Get secret list by virsh secret-list from local or remote host.
+
+    :param remote_virsh: remote virsh shell session.
+    :return secret list
+    """
+    logging.info("Get secret list ...")
+    if remote_virsh:
+        secret_list_result = remote_virsh.secret_list()
+    else:
+        secret_list_result = virsh.secret_list()
+    secret_list = secret_list_result.stdout_text.strip().splitlines()
```

Comment:
it is good to wrap it in `try..` block and catch any obvious error outputs and return proper result incase failure condition.

Reply:
updated

Reply:
updated